DSL.Ruuter/POST/pseudonymise.yml

```diff
@@ -0,0 +1,14 @@
+pseudonymise:
+  call: http.post
+  args:
+    url: http://web:5001/predict
```

Comment:
What endpoint is `http://web:5001/predict`?

Reply:
This is the endpoint which is hosted on our Anonymiser Service. It pseudonymises a particular text using the trained model.